app/delivery/send_to_providers.py

```diff
@@ -214,6 +228,7 @@ def provider_to_use(notification: Notification):
         )
         raise Exception("No active {} providers".format(notification.notification_type))
```

Comment:
small quibble, to be consistent with `app/clients/sms/twilio.py` on line 103 above, can we use f strings here for the error and exception messages, instead of .format() ?

Reply:
Yeah, there's a lot of places I'd like to change to using f strings instead of .format(). I've just been slowly changing some of them here and there as I work on different files.